react-ui/src/foundwork/index.js

```diff
@@ -1,7 +1,53 @@
+import React from 'react';
+import {
+	LineChart,
+	Line,
+	XAxis,
+	YAxis,
+	Tooltip,
+	CartesianGrid,
+	Legend,
+} from 'recharts';
+
 const FoundWork = ({ foundWorkData }) => {
+    console.log(foundWorkData)
+    const tasksByDate = foundWorkData?.reduce((acc, task) => {
+        const date = new Date(task.createdDate).toDateString();
+        acc[date] = (acc[date] || 0) + 1;
+        return acc;
+    }, {});
+    const chartData = Object.entries(tasksByDate).map(([date, count]) => ({
+        date,
+        count,
+    }));    
     return(
-        <div>
-            <p>{foundWorkData.length}</p>
+        <div align="center" width={750} height={300} sx={{ mt: 2 }}>
```

Comment:
You can keep it in **ResponsiveContainer** with width 100% and height 600.

Also, add a haeding at the top.

For reference: https://github.com/shahih25/SER516-Team-Clever/blob/main/react-ui/src/graph/linechart.js#L24